package.json

```diff
@@ -27,20 +31,38 @@
   "author": "Ben Coe <ben@npmjs.com>",
   "license": "ISC",
   "devDependencies": {
+    "@types/chai": "^4.2.11",
+    "@types/mocha": "^7.0.2",
+    "@types/node": "^10.0.3",
+    "@typescript-eslint/eslint-plugin": "^2.25.0",
+    "@typescript-eslint/parser": "^2.25.0",
     "c8": "^7.0.1",
     "chai": "^4.2.0",
+    "eslint": "^7.0.0",
+    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-node": "^11.0.0",
+    "gts": "^2.0.0-alpha.4",
     "mocha": "^7.0.0",
-    "standard": "^14.3.1"
+    "rimraf": "^3.0.2",
+    "standardx": "^5.0.0",
+    "typescript": "^3.7.0"
   },
   "dependencies": {
     "camelcase": "^5.0.0",
     "decamelize": "^1.2.0"
   },
   "files": [
-    "lib",
-    "index.js"
+    "index.js",
+    "build",
+    "lib/**/*.js"
   ],
   "engines": {
     "node": ">=6"
```

Comment:
If we do want to continue supporting node 6 in `yargs-parser`, then the target needs to be changed in `tsconfig.json` (and in `.eslintrc` too, I think), as node 6 supports only 17% of es2017 (see https://node.green/#ES2017).

Reply:
But as I see we no longer do tests on node 6, I suggest upgrading the engines requirements to `"node": ">=8"` here

Reply:
https://github.com/yargs/yargs-parser/pull/278

Reply:
OK, but opening #278 does not change the issue here: tsconfig.json and .eslintc are still incoherent with package.json?

Reply:
@mleguen engines.node in package.json has been updated to `>=10`, ESLint `parserOptions.ecmaVersion` updated to `2018`, and updated tsconfig.json to remove `es2017` so it will use `es2018` from `gts/tsconfig-google.json`.